src/CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ set_property(TARGET libpkgmanifest PROPERTY POSITION_INDEPENDENT_CODE ON)
 install(TARGETS libpkgmanifest LIBRARY DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR})
 
 list(APPEND libpkgmanifest_PC_REQUIRES_PRIVATE "${YAMLCPP_MODULE_NAME}")
+list(APPEND libpkgmanifest_PC_REQUIRES_PRIVATE "${RPM_MODULE_NAME}")
 target_link_libraries(libpkgmanifest PRIVATE ${YAMLCPP_LIBRARIES})
+target_link_libraries(libpkgmanifest PRIVATE ${RPM_LIBRARIES})
```

Comment:
I worry that RPM_LIBRARIES only contains -l flags. What about -L flags? What about CFLAGS (e.g. rpm pkg-config defines -D_FILE_OFFSET_BITS=64)? I think there are more variables <https://cmake.org/cmake/help/latest/module/FindPkgConfig.html> you should use. Maybe IMPORTED_TARGET feature can handle all these variables automatically.

Reply:
That sounds reasonable.  I think I did it right in the update I made.